Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -6319,7 +6319,7 @@ TEST_CASE("MaeWriter basic testing", "[mae][MaeWriter][writer]") {
 
     std::string_view ctBlock(&mae[ctBlockStart]);
 
-    CHECK(ctBlock == MaeWriter::getText(*mol));
+    CHECK((ctBlock == MaeWriter::getText(*mol)) == true);
```

Comment:
I will remove the change, but without it the system does not build.  There is a problem between string_view and Catch2.

Reply:
> I will remove the change, but without it the system does not build. There is a problem between string_view and Catch2.

This does not happen for me on any machine I build the RDKit on, nor does it happen on any of the CI systems. This is likely a problem with your local configuration

Reply:
This happened to me also, on my Macs.  It seems that it is because Catch2 needs to be compiled with C++17 compatibility in order to use string_view and by default its build uses C++14.  I deleted my installation in /usr/local and RDKit built a version that worked for me.